README.rst

```diff
@@ -57,8 +57,10 @@ Releasing
 3. Add a commit which describes the changes from the previous version to ``CHANGES.rst`` and updates the version number in ``lib/nextstrain/sphinx/theme/VERSION``.
 4. Tag this commit with the version number, e.g. ``git tag -a 2020.4 -m "version 2020.4"``.
 5. Push the commit and tag to GitHub, e.g. ``git push origin main 2020.4``.
-6. `Generate the source and binary distributions with setup.py <https://packaging.python.org/tutorials/packaging-projects/#generating-distribution-archives>`__.
-7. `Upload the two files from step 6 to PyPI with twine <https://packaging.python.org/tutorials/packaging-projects/#uploading-the-distribution-archives>`__.
+6. Publish to PyPI by invoking a GitHub Actions workflow:
+
+    1. Go to the workflow: `publish.yml <https://github.com/nextstrain/sphinx-theme/actions/workflows/publish.yml>`_.
+    2. Select **Run workflow** > **Run workflow**.
```

Comment:
Important to specify the tag you just pushed when triggering the workflow, otherwise there's a race condition here where something else on `main` could get released.

Reply:
Added: 1d1522a

Reply:
Ah, so I meant in the instructions here, not as a new workflow input. Since workflows always run against a specific ref, the UI for making a `workflow_dispatch` event already lets you choose it without adding an input.

If you haven't seen what manually dispatching a workflow looks like, here's an example screenshot [from the docs](https://docs.github.com/en/actions/managing-workflow-runs/manually-running-a-workflow#running-a-workflow):

![](https://docs.github.com/assets/cb-22017/images/actions-manually-run-workflow.png)

The "Use workflow from" input is implicit/automatic.

Reply:
Ohh that's neat, I didn't realize there was an option to choose tag over branch. Will update.

Reply:
Done in recent force-pushes.